src/localization/strings.ts

```diff
@@ -0,0 +1,36 @@
+import en from "./en.json";
+import fi from "./fi.json";
+import LocalizedStrings, { LocalizedStringsMethods } from "localized-strings";
+
+/**
+* Language names interface
+*/
+interface Localizations {
+  en: string;
+  fi: string;
+}
+
+/**
+* Translations interface
+*/
+interface Translations {
+  notYetImplemented: string;
+  currentLocaleLabel: string;
+  localization: Localizations;
```

Comment:
Typically the localization would be defined in the main interface Translations (which is typically called Localized) rather than as a separate interface.